gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -35,7 +35,7 @@
 #define NNS_TENSOR_RANK_LIMIT_PREV (4)
 #define NNS_TENSOR_SIZE_LIMIT	(16)
 #define NNS_TENSOR_SIZE_LIMIT_STR	"16"
-#define NNS_TENSOR_SIZE_EXTRA_LIMIT (100)
+#define NNS_TENSOR_SIZE_EXTRA_LIMIT (240)
```

Comment:
Suggestion: how about updating NNS_TENSOR_SIZE_LIMIT (256)?

e.g.,
NNS_TENSOR_SIZE_LIMIT_STATIC (16)
NNS_TENSOR_SIZE_EXTRA_LIMIT (NNS_TENSOR_SIZE_LIMIT - NNS_TENSOR_SIZE_LIMIT_STATIC)

Reply:
You're right. That is obviously proper way of defining the value. I added a todo comment.
I'll do the work after the ACR of ml-api is done.